### ISSUE TITLE
[Help] [Bug?]

### ISSUE THREAD

**Issue by sarahsimionescu** [oleeskild/obsidian-digital-garden#168]
Example of bug is on my site: https://sarahsdigitalgarden.netlify.app/
My github repo with my site: https://github.com/sarahsimionescu/digitalgarden

Many but not all of my front links do not work. Also the file navigation does not work either for some reason, let me know if this is a bug or if I am doing something wrong!

I thought the cause could be a lack of support for the vertical pipe command [[Article#Header|Put text here]] which I use quite often, but it seems like there are some links that work.

Please let me know if you know how to fix this! Thank you!

**Comment by oleeskild** [comment]
There seems to be an issue when the plugin is processing your note before uploading it to github 🤔 

[This file](https://github.com/sarahsimionescu/digitalgarden/blob/main/src/site/notes/COMPSCI%202XC3/0.5%20Introduction/Recursion%20vs%20Iteration.md?plain=1) for example, **should** look something like this:
```
---
{"dg-publish":true,"permalink":"/compsci-2-xc-3/0-5-introduction/recursion-vs-iteration/"}
---

**Recursive implementations (generally) take longer to run the iterative implementations.**
```

I'm not sure why this happens. What platform are you on? Windows, Mac, Linux, iOS or Android?

**Comment by sarahsimionescu** [comment]
Hello! Sorry for this late reply, I had a busy weekend. Thank you so much for your feedback. I am using Windows. Any advice on how to fix the issue?

**Comment by oleeskild** [comment]
No worries!
I’m suspecting it may have something to do with the some of the folder names. Can you try creating a copy of the note I mentioned above in the root of your vault, and publish that? Let me know when you’ve done that, and I’ll have a look in your repo to see if it looks correct or not.

**Comment by oleeskild** [comment]
So that note seem to have been correctly added. This could suggest that this has something to do with how Windows handles folder names or something. I tried replicating the issue myself, but I couldn't. But I'm on a Mac, so I need to see if I can find a Windows computer to test this. 

Out of curiosity: Does any of the files show up under the "Changed" section in the Publication Center, even if you've published them?

**Comment by sarahsimionescu** [comment]
Nope, the Publication Center works as expected. Thank you so much for your time.

**Comment by sarahsimionescu** [comment]
Everything suddenly works! Thank you!

**Comment by oleeskild** [comment]
Oh, okay 😅 Great 👍
